server/src/main/java/org/elasticsearch/Version.java

```diff
@@ -176,10 +176,9 @@
     public static final Version V_6_3_2 = new Version(V_6_3_2_ID, org.apache.lucene.util.Version.LUCENE_7_3_1);
     public static final int V_6_4_0_ID = 6040099;
     public static final Version V_6_4_0 = new Version(V_6_4_0_ID, org.apache.lucene.util.Version.LUCENE_7_4_0);
-    public static final int V_7_0_0_alpha1_ID = 7000001;
-    public static final Version V_7_0_0_alpha1 =
-        new Version(V_7_0_0_alpha1_ID, org.apache.lucene.util.Version.LUCENE_7_4_0);
-    public static final Version CURRENT = V_7_0_0_alpha1;
+    public static final int V_7_0_0_ID = 7000001;
+    public static final Version V_7_0_0 = new Version(V_7_0_0_ID, org.apache.lucene.util.Version.LUCENE_7_4_0);
+    public static final Version CURRENT = V_7_0_0;
```

Comment:
This requires some thought as this now depends on the build. I think that we might need to dynamically ascertain the value of `CURRENT` based on the version and the qualifier (dynamically compute the ID).

Reply:
If we do that, we'll have to model the qualifier here again. 
My understanding is that this code will not know about the qualifier,
in this respect I don't think there's any change, we used to have the version and qualifier hard coded here with matching in `version.properties`, so if we no longer care about the qualifier here, 
it pretty much stays the same regardless of the fact that the qualifier can be changed at build time.

Reply:
@jasontedor re-reading your comment,  I think I  misinterpreted it. Are you thinking about determining CURRENT based on the version in the jar ? Topically we would change this for an alpha and will not have to update it after that. We could make a change in a separate PR to assign this based on the manifest. let me know.